packages/grafbase-sdk/README.md

```diff
@@ -307,4 +307,36 @@ const user = g.model("User", {
   name: g.string(),
   age: g.int()
 }).search()
-```
\ No newline at end of file
+```
+
+## Connectors
+
+Connectors are created through the connector interface:
+
+```typescript
+import { connector } from '../../src/index'
+```
+
+### OpenAPI
+
+The OpenAPI connector can be created with the `OpenAPI` method:
+
+```typescript
+const openai = connector.OpenAPI({
+  schema: 'https://raw.githubusercontent.com/openai/openai-openapi/master/openapi.yaml'
+})
+
+const stripe = connector
+  .OpenAPI({
+    schema: 'https://raw.githubusercontent.com/stripe/openapi/master/openapi/spec3.json',
+    url: 'https://api.stripe.com'
+  })
+  .header('Authorization', 'Bearer {{ env.STRIPE_API_KEY }}')
```

Comment:
Should header be builder or another key inside OpenAPI?

Reply:
Right, but then we have some attributes that are set via build and some that aren't. I like the consistency of having everything related to the connector inside the {} personally.